src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIWaveEquationSEM.cpp

```diff
@@ -0,0 +1,871 @@
+  {
+    NodeManager & nodeManager = mesh.getNodeManager();
+
+    arrayView1d< real32 const > const mass = nodeManager.getField< fields::MassVector >();
+    arrayView1d< real32 const > const damping_p = nodeManager.getField< fields::DampingVector_p >();
+    arrayView1d< real32 const > const damping_q = nodeManager.getField< fields::DampingVector_q >();
+    arrayView1d< real32 const > const damping_pq = nodeManager.getField< fields::DampingVector_pq >();
+    arrayView1d< real32 const > const damping_qp = nodeManager.getField< fields::DampingVector_qp >();
+
+    arrayView1d< real32 > const p_nm1 = nodeManager.getField< fields::Pressure_p_nm1 >();
+    arrayView1d< real32 > const p_n = nodeManager.getField< fields::Pressure_p_n >();
+    arrayView1d< real32 > const p_np1 = nodeManager.getField< fields::Pressure_p_np1 >();
+
+    arrayView1d< real32 > const q_nm1 = nodeManager.getField< fields::Pressure_q_nm1 >();
+    arrayView1d< real32 > const q_n = nodeManager.getField< fields::Pressure_q_n >();
+    arrayView1d< real32 > const q_np1 = nodeManager.getField< fields::Pressure_q_np1 >();
+
+    arrayView1d< localIndex const > const freeSurfaceNodeIndicator = nodeManager.getField< fields::FreeSurfaceNodeIndicator >();
+    arrayView1d< localIndex const > const lateralSurfaceNodeIndicator = nodeManager.getField< fields::LateralSurfaceNodeIndicator >();
+    arrayView1d< localIndex const > const bottomSurfaceNodeIndicator = nodeManager.getField< fields::BottomSurfaceNodeIndicator >();
+    arrayView1d< real32 > const stiffnessVector_p = nodeManager.getField< fields::StiffnessVector_p >();
+    arrayView1d< real32 > const stiffnessVector_q = nodeManager.getField< fields::StiffnessVector_q >();
+    arrayView1d< real32 > const rhs = nodeManager.getField< fields::ForcingRHS >();
+
+    //DEbug
+    arrayView1d< real32 > const stiffnessVector = nodeManager.getField< fields::StiffnessVector >();
+    arrayView1d< real32 > const damping = nodeManager.getField< fields::DampingVector >();
+
+    auto kernelFactory = acousticVTIWaveEquationSEMKernels::ExplicitAcousticSEMFactory( dt );
+
+    finiteElement::
+      regionBasedKernelApplication< EXEC_POLICY,
+                                    constitutive::NullModel,
+                                    CellElementSubRegion >( mesh,
+                                                            regionNames,
+                                                            getDiscretizationName(),
+                                                            "",
+                                                            kernelFactory );
+
+    addSourceToRightHandSide( cycleNumber, rhs );
+
+    /// calculate your time integrators
+    real64 const dt2 = dt*dt;
+
+//    if( !usePML )
+  //  {
+      GEOS_MARK_SCOPE ( updateP );
+      forAll< EXEC_POLICY >( nodeManager.size(), [=] GEOS_HOST_DEVICE ( localIndex const a )
+      {
+        //Debug
```

Comment:
Remove commented section. Also the check for PMLs, since they are not implemented for this solver

Reply:
Comment removed. PML functions are kept but do nothing except throwing an error.